static/js/constants.js

```diff
@@ -12,6 +12,8 @@ export const DASHBOARD_FORMAT = 'MMM D, Y';
 export const DASHBOARD_MONTH_FORMAT = 'MM[/]YYYY';
 
 export const CP1252_REGEX = /^[\u0020-\u00FF]*$/;
+// greater-than, comma, quotation-mark
```

Comment:
It's a code comment which is programmer-facing so I'm inclined to leave it as-is because it's more succinct.